assume/strategies/extended.py

```diff
@@ -2,8 +2,11 @@
 #
 # SPDX-License-Identifier: AGPL-3.0-or-later
 
+import dateutil.rrule as rr
```

Comment:
why the name is extended? maybe a more specific name would be better?

Reply:
Sure - but as it was that way before, I would leave this as is in this PR and change it once we have a proposal?
I don't know a much better name - it just includes some not basic strategies..